tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py27-django{18,111}
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE = eventtracking.django.tests.settings
+    PYTHONPATH = {toxinidir}
+
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/dev-requirements.txt
```

Comment:
@bmedx It took me a while to figure out what was going on with help-tokens and django-user-tasks.  There's actually a sed call to remove the django requirement in test.txt in the Makefile.  I guess the use of pip-compile in these repos helps to address some of the cut-and-paste concerns that I had.  For django-user-tasks, I was also thrown off by the lack of a Django requirement at all in setup.py.  I'm thinking this would be cleaner by just removing the tox-requirements.txt file and putting the Django dependency in setup.py rather than a separate file that is only used there.